### DISCUSSION TITLE
swagger Error

### DISCUSSION THREAD

**Discussion by sriniazhagar** [JanssenProject/jans#6087] [Q&A] [answered]
![image](https://github.com/JanssenProject/jans/assets/129082509/91a9c420-b2cc-4862-b98f-7a8a5f2a017a)
While passing the https://raw.githubusercontent.com/JanssenProject/jans/v1.0.17/jans-auth-server/docs/swagger.yaml file to swagger.

We are getting the Errors like 
"Errors Hide Parser error on line 2528 bad indentation of a mapping 
entry Errors "

Content on the Line no 2528 is 
description: Client Id. It can be useful to specify client id explicitly in case id_token and session are expired so AS can still validate "post_logout_redirect_uri"

Need support on the error closure.

**Comment by yuriyz** [comment, marked as answer]
It's fixed here https://github.com/JanssenProject/jans/pull/6086

Valid version is in main: https://raw.githubusercontent.com/JanssenProject/jans/main/jans-auth-server/docs/swagger.yaml
